authors/apps/authentication/tests/test_validation.py

```diff
@@ -0,0 +1,68 @@
+from django.test import TestCase
+from rest_framework.test import APIClient
+from rest_framework import status
+from authors.apps.authentication.models import User
+from django.core import mail
+from rest_framework.test import APIRequestFactory
+from django.urls import reverse
+from authors.apps.authentication.backends import generate_jwt_token
+
+
+class VerifyTestCase(TestCase):
+    """
```

Comment:
@Bileonaire after reviewing PEP8 style guide, how you had kept the closing triple double-quotes was correct on the same line was correct e.g """ My docstring """. 
From PEP8 style guide: "For one liner docstrings, please keep the closing """ on the same line"